setup.py

```diff
@@ -69,4 +71,5 @@ def get_version():
         "fapolicy_analyzer.resources.sourceview.language-specs": ["*.lang"],
         "fapolicy_analyzer.resources.sourceview.styles": ["*.xml"],
     },
+    cmdclass={"build_help": build_help},
```

Comment:
What was the advantage of making build_help a setup cmd vs. just a plain old script?

Reply:
Well the original thought way back when was this would run as part of the of RPM build like the `compile_catalog` command. The nice thing about the current setup is a developer doing translations or running locally just needs to run `python setup.py build_help` and they will have a copy of the help locally. Are you thinking we make this a standalone script?

Reply:
> @jw3 can you handle updating the RPM build process?

Yeah looking now.

Can you show a quick example of

1. Call the script that clones and downloads the images
2. Call the `build_help` using the files from (1)

Reply:
Sure. 
1. To download a local copy of the help file and convert it to HTML call `pipenv run python setup.py update_help`. The defaults will place it under the project root in the `help` directory. This will also create the portable translation file that can be used for doing translations.
    - If you need to pull the help for a specific commit update the commit in the `help/.env` file and export `export PIPENV_DOTENV_LOCATION=help/.env ` or use the `-c` flag when calling the command giving the commit number.
    - If you need to use a proxy server for downloading use the `-p` flag when calling the command giving the URL of the proxy server.
2. To build the help for release, run `pipenv run python setup.py build_help`. By default this will use the `help` directory as source and output to `build/help`. This also performs any translations and places the translated files in the output.

Reply:
Thanks.  Sounds good.  The path forward is

1. Run step 1 outside the RPM build, tar up the results, pass in as SourceX
2. Run step 2 in the RPM build and install the results

I'd forgot about the .mo previously.  We cant have binaries in the srpm.

I'll update the RPM build.

Reply:
@dorschs57  Can you give me a quick example of doing `update_help` without going through setup.py?

Earlier I overlooked an important point.. that we also need to decouple the help doc download from the setuptools-rust dependency (which is not available on RHEL).  As long as the download call is made from setup.py this cant happen.

Reply:
@dorschs57 

I tried several ways to run the update outside of setup.py, got the clone but not the conversion.  That logic down in the command class couldnt be hijacked with the time available right now.

My preference is to convert the commands into standalone scripts and call them from the Makefile rather than setup.py.  If that is possible I think we can still give the user a simple way to build all this stuff, eg `make docs` and avoid the issues with setup.py dependencies.

Reply:
@jw3  I updated the setup cmds to a standalone script called `help_utils.py`. You should now be able to run `python help_utils.py update` to update and `python help_utils.py build` to build and I think the defaults will work. Again the commit can be changed by updating the hash in the `help/.env` file.